components/desktop/gnome2/gnome-connection-manager/Makefile

```diff
@@ -10,34 +10,48 @@
 #
 
 #
-# Copyright (c) 2015 Jim Klimov
+# Copyright (c) 2015-2020 Jim Klimov
 #
 
 include ../../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME=		gnome-connection-manager
+COMPONENT_NAME_LONG=	gnome-connection-manager
 COMPONENT_NAME_SHORT=	gcm
-COMPONENT_VERSION=	1.1.0
+# The value below should match the .p5m and .license filenames, in particular:
+COMPONENT_NAME=  		$(COMPONENT_NAME_LONG)
+COMPONENT_VERSION=	1.2.0
 COMPONENT_REVISION=	1
 COMPONENT_PROJECT_URL=	http://kuthulu.com/gcm
-COMPONENT_SRC=		$(COMPONENT_NAME_SHORT)-$(COMPONENT_VERSION)
+# The new GCM release should bring support for Gtk3 and Python3,
+# and is expected to work in OI unlike the venerable 1.1.0 made
+# for Gtk2 which just stopped working. So far it is unreleased,
+# hence the GitHub latest-commit fetching and tweaked definitions
+# of COMPONENT_SRC and COMPONENT_ARCHIVE below:
+COMPONENT_COMMIT_ID=	878d59ed3a77db55a6d22bbda5ce1ee1acac16a5
+COMPONENT_SRC=		$(COMPONENT_NAME_LONG)-$(COMPONENT_COMMIT_ID)
+###COMPONENT_SRC=		$(COMPONENT_NAME_SHORT)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE_HASH=	\
-    sha256:71a128676d16153c735499cec9f0a1af1e2dc1107ed5c57cea1994eadc120c4c
+    sha256:92f0db50a31cee9bce271a3f85c269d604737edf139e0b55d52a36c0217876c1
 # Project site is gone, use a mirror for now
-#COMPONENT_ARCHIVE=	$(COMPONENT_NAME)_$(COMPONENT_VERSION)_all.tgz
+#COMPONENT_ARCHIVE=	$(COMPONENT_NAME_LONG)_$(COMPONENT_VERSION)_all.tgz
 #COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)/$(COMPONENT_ARCHIVE)
-COMPONENT_ARCHIVE=	$(COMPONENT_NAME)_$(COMPONENT_VERSION).orig.tar.gz
-COMPONENT_ARCHIVE_URL= http://ftp.heanet.ie/mirrors/www.getdeb.net/getdeb/ubuntu/pool/apps/g/$(COMPONENT_NAME)/$(COMPONENT_ARCHIVE)
+###COMPONENT_ARCHIVE=	$(COMPONENT_NAME_LONG)_$(COMPONENT_VERSION).orig.tar.gz
+COMPONENT_ARCHIVE=	$(COMPONENT_COMMIT_ID).zip
```

Comment:
Please, name COMPONENT_ARCHIVE as $(COMPONENT_NAME)-$(COMPONENT_COMMIT_ID).zip, this determines how this file will be stored.
You can modify COMPONENT_ARCHIVE_URL to use $(COMPONENT_COMMIT_ID).zip unrelated to this.